admin/admin-domain-service/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'net.researchgate.release' version '3.0.2' apply false
+}
+
+subprojects {
+    task bal_build(type: Exec) {
+        group 'ballerina'
+        description 'Build ballerina code'
+        commandLine 'bal', 'build'
+    }
+    
+    task bal_test(type: Exec) {
+        group 'ballerina'
+        description 'Run ballerina tests'
+        commandLine 'bal', 'test'
+    }
```

Comment:
shall we create common ballerina.gradle in common-gradle-scripts and import it here like below?

`apply from: "$rootDir/../common-gradle-scripts/ballerina.gradle"`

Reply:
Fixed with [ca3ec70](https://github.com/wso2/apk/pull/138/commits/ca3ec702a70164c55f4f85fed7694b02286dc96c)